tests/test_optimizer.py

```diff
@@ -983,6 +984,141 @@ def test_constrained_AL_scalar():
     np.testing.assert_array_less(-Dwell, ctol)
 
 
+@pytest.mark.slow
+@pytest.mark.unit
+@pytest.mark.optimize
+def test_proximal_with_PlasmaVesselDistance():
```

Comment:
is this test new? I thought we already had one like this?

Reply:
Not new, I think you might have removed it in your test cleanup PR though. Probably this is unneeded and I can remove it now as I assume elsewhere something similar tests it